device/broadcom/x86_64-broadcom_common/syncd_shm.ini

```diff
@@ -0,0 +1,14 @@
+# This file contains the default memory size for each ASICs in broadcom platform
+# Format: ASIC_ID=Memory_size
+#b77->td3
+b77=256m
+#b85->td2
+b85=256m
+#b87->td3
+b87=512m
+#b96->th
+b96=256m
+#b97->th2
+b97=256m
+#b98->th3
+b98=300m
```

Comment:
these are all xgs chips, what about dnx chips? do we need to set different memory size for them like jr2, jr2c?

Reply:
Current memory set is only for xgs chips, not include the dnx chips IDs.